mqtt/mqtt-edgehub/src/auth/authorization/edgehub.rs

```diff
@@ -140,34 +140,14 @@ fn is_iothub_topic(topic: &str) -> bool {
 
 fn allowed_iothub_topic(client_id: &ClientId) -> Vec<String> {
     vec![
-        format!("$edgehub/clients/{}/messages/events", client_id),
-        format!("$iothub/clients/{}/messages/events", client_id),
-        format!("$edgehub/clients/{}/messages/c2d/post", client_id),
-        format!("$iothub/clients/{}/messages/c2d/post", client_id),
-        format!(
-            "$iothub/clients/{}/twin/patch/properties/desired",
-            client_id
-        ),
-        format!(
-            "$edgehub/clients/{}/twin/patch/properties/desired",
-            client_id
-        ),
-        format!(
-            "$iothub/clients/{}/twin/patch/properties/reported",
-            client_id
-        ),
-        format!(
-            "$edgehub/clients/{}/twin/patch/properties/reported",
-            client_id
-        ),
-        format!("$edgehub/clients/{}/twin/get", client_id),
-        format!("$iothub/clients/{}/twin/get", client_id),
-        format!("$edgehub/clients/{}/twin/res", client_id),
-        format!("$iothub/clients/{}/twin/res", client_id),
-        format!("$edgehub/clients/{}/methods/post", client_id),
-        format!("$iothub/clients/{}/methods/post", client_id),
-        format!("$edgehub/clients/{}/methods/res", client_id),
-        format!("$iothub/clients/{}/methods/res", client_id),
+        format!("$edgehub/{}/messages/events", client_id),
```

Comment:
Just to confirm: we were using $iothub/$edgeHub to distinguish which topics need to be "translated" and which don't. We don't have $iothub anymore because we don't have any clients using those topics, right? 
Once we support the nested scenario with the bridge, we will have to support that?

Reply:
Correct. Initially, I went ahead and "implemented" checks for a nested scenario which turned out to be done prematurely. Will return to it once mqtt topics are stabilized